tests/ui/test_ui.py

```diff
@@ -249,29 +253,31 @@ def test_keypress_autohide_streams(self, view, mocker, autohide, key):
         size = (20,)
 
         super_view = mocker.patch("zulipterminal.ui.urwid.WidgetWrap.keypress")
-        view.controller.editor_mode = False
+        view.controller.is_in_editor_mode = lambda: False
 
         view.body.focus_position = None
 
         view.keypress(size, key)
+
         view.left_panel.keypress.assert_called_once_with(size, key)
         assert view.body.focus_position == 0
         (view.stream_w.stream_search_box.set_edit_text
          .assert_called_once_with(""))
-        assert view.controller.editor_mode is True
-        assert view.controller.editor == view.stream_w.stream_search_box
+        (view.controller.enter_editor_mode_with
+         .assert_called_once_with(view.stream_w.stream_search_box))
 
     @pytest.mark.parametrize('key', keys_for_command('SEARCH_PEOPLE'))
     def test_keypress_edit_mode(self, view, mocker, key):
         view.users_view = mocker.Mock()
         view.body = mocker.Mock()
         view.user_search = mocker.Mock()
-        size = (20,)
```

Comment:
Move this dead code to a separate commit?

Reply:
I think there's just this one line and a comment? That seems rather minor to have separately?